### ISSUE TITLE
upgdare ojs 3.3 to 3.4

### ISSUE THREAD

**Issue by mjamilo** [pkp/pkp-lib#9903]
## 
Next Illuminate\Database\QueryException: SQLSTATE[HY000]: General error: 1364 Field ‘setting_type’ doesn’t have a default value (SQL: insert into user_settings (user_id, locale, setting_name, setting_value) values (2, , orcid, a:1:{s:2:“en”;s:0:“”;})) in /home/univgres/journal.www/lib/pkp/lib/vendor/laravel/framework/src/Illuminate/Database/Connection.php:760
Stack trace:
#0 /home/uni/journal.www/lib/pkp/lib/vendor/laravel/framework/src/Illuminate/Database/Connection.php(720): Illuminate\Database\Connection->runQueryCallback()
#1 /home/uni/journal.www/lib/pkp/lib/vendor/laravel/framework/src/Illuminate/Database/Connection.php(546): Illuminate\Database\Connection->run()
#2 /home/uni/journal.www/lib/pkp/lib/vendor/laravel/framework/src/Illuminate/Database/Connection.php(498): Illuminate\Database\Connection->statement()
#3 /home/uni/journal.www/lib/pkp/lib/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(3274): Illuminate\Database\Connection->insert()
#4 /home/uni/journal.www/lib/pkp/lib/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(3392): Illuminate\Database\Query\Builder->insert()
#5 /home/uni/journal.www/lib/pkp/classes/core/EntityDAO.php(239): Illuminate\Database\Query\Builder->updateOrInsert()
#6 /home/uni/journal.www/lib/pkp/classes/user/DAO.php(270): PKP\core\EntityDAO->_update()
#7 /home/uni/journal.www/lib/pkp/classes/user/Repository.php(121): PKP\user\DAO->update()
#8 /home/uni/journal.www/lib/pkp/classes/security/Validation.php(158): PKP\user\Repository->edit()
#9 /home/uni/journal.www/lib/pkp/classes/security/Validation.php(79): PKP\security\Validation::registerUserSession()
#10 /home/uni/journal.www/lib/pkp/pages/login/LoginHandler.php(149): PKP\security\Validation::login()
#11 [internal function]: PKP\pages\login\LoginHandler->signIn()
#12 /home/uni/journal.www/lib/pkp/classes/core/PKPRouter.php(334): call_user_func()
#13 /home/uni/journal.www/lib/pkp/classes/core/PKPPageRouter.php(277): PKP\core\PKPRouter->_authorizeInitializeAndCallRequest()
#14 /home/uni/journal.www/lib/pkp/classes/core/Dispatcher.php(165): PKP\core\PKPPageRouter->route()
#15 /home/uni/journal.www/lib/pkp/classes/core/PKPApplication.php(388): PKP\core\Dispatcher->dispatch()
#16 /home/uni/journal.www/index.php(21): PKP\core\PKPApplication->execute()
#17 {main}
thrown in /home/uni/journal.www/lib/pkp/lib/vendor/laravel/framework/src/Illuminate/Database/Connection.php on line 760

help me…thanks

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

**Comment by jonasraoni** [comment]
Hi @mjamilo! Looks like you didn't upgrade the system properly, just updating the files isn't enough, please refer to: https://docs.pkp.sfu.ca/dev/upgrade-guide/en/
